### PR TITLE
Added tool Xprobe2 and contributor

### PR DESCRIPTION
**Pull request by SAAneeraj**
Added tool Xprobe2 and contributor

### CODE REVIEW

**Comment by SAAneeraj** [comment]
Hope this PR counts for hacktober fest, else please guide me.

File: data/tools/networking.json [hunk 1]
```diff
@@ -609,6 +609,14 @@
       "price": "Free",
       "online": "False"
     },
+    { 
+      "name": "Xprobe2",
+      "source": "https://github.com/binarytrails/xprobe2.git",
```
Comment (noraj) [inline data/tools/networking.json:614]: please remove the `.git`

File: data/tools/networking.json [hunk 2]
```diff
@@ -609,6 +609,14 @@
       "price": "Free",
       "online": "False"
     },
+    { 
+      "name": "Xprobe2",
+      "source": "https://github.com/binarytrails/xprobe2.git", 
+      "description": "Xprobe2 is a remote active operating system fingerprinting tool.",
```
Comment (noraj) [inline data/tools/networking.json:615]: See [this](https://inventory.raw.pm/docs/docs/en/add/) and [this](https://inventory.raw.pm/docs/docs/en/format/)

- No final dot
- No tool name

File: data/tools/networking.json [hunk 3]
```diff
@@ -609,6 +609,14 @@
...
+      "language": "C++",
```
Comment (noraj) [inline data/tools/networking.json:616]: change to `CPlusPlus`

**Review by noraj — CHANGES_REQUESTED** [review]
Hi, thank you a lot for your contribution. Can you please make the few required changes?

File: CONTRIBUTORS.txt [hunk 4]
```diff
@@ -12,3 +12,4 @@ Content and improvements:
     E99p1ant (@wuhan005)
     Jenaye (@jenaye)
     Maraudery (@maraudery)
+    Neeraj Babu (@SAAneeraj)
```
Comment (noraj) [inline CONTRIBUTORS.txt:15]: You can also add your contributor badge to https://github.com/noraj/rawsec-cybersecurity-inventory/blob/master/README.md#contributions

Comment (SAAneeraj) [reply CONTRIBUTORS.txt:15]: Sry I did'nt really understand it can you please guide me

Comment (SAAneeraj) [reply CONTRIBUTORS.txt:15]: I figured it out and made another pull request to main branch

**Review by noraj — APPROVED** [review]

**Pull request merged**